luaui/RmlWidgets/gui_awards/gui_awards.rcss

```diff
@@ -0,0 +1,171 @@
+* {
+    font-family: "Exo 2";
+    font-size: 18px;
+    color: white;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+  background-color: rgba(255,255,255,0.7)
+}
+
+table {
```

Comment:
Good on you for leveraging semantic elements, but we are not in the browser and as such they mean nothing, there is no web-crawler trying to understand the document for SEO reasons. Use divs with clearly defined properties. Ordinarily we might try and piggy-back on default layout and styles of elements in the browser but in rmlui you can't rely on what default styles even exist. Plus the data you are displaying here is not tabular, ie clear rows and columns and as such, again, just use divs.